src/ErrorHandler.php

```diff
@@ -337,4 +297,32 @@ private function cleanBacktraceFromErrorHandlerFrames(array $backtrace, string $
 
         return $cleanedBacktrace;
     }
+
+    /**
+     * @param \ErrorException $errorAsException The error to be passed onto listeners
+     */
+    private function invokeErrorListeners(\ErrorException $errorAsException): void
```

Comment:
Since both the error and exception listeners have the same signature (they accept an exception instance) wouldn't it make sense to have just one method that iterates over a given parameter containing the list of callbacks? So instead of just the exception we also pass to this method the array of listeners to invoke

Reply:
Good suggestion. Will refactor.